include/openassetio/managerAPI/ManagerInterface.hpp

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+#pragma once
+
+#include <openassetio/abi.hpp>
```

Comment:
Where is this file?

I'm also used to seeing an "API" header, e.g. for visibility attributes and declspecs. A little surprised to see an ABI header, but curious to see what it contains.

Reply:
Yeah I've got a bit inconsistent with the naming, especially when using other repos as a reference.  The API header and ABI header are one and the same.  `abi.hpp` is what is created by `generate_export_header`.

Any opinions on a consistent naming scheme for such a header and reference to it in text?  I've provisionally settled on "ABI" since it has a more specific meaning than "API".

Reply:
Oh it's generated. I hadn't put two and two together.
As `generate_export_header` writes #defines to do with symbol export, this I feel is API.
So I would name it api.h or export.h.

ABI feels more than just defining the surface API visibility.

I've never seen an abi header with that content before.

Reply:
Renamed to `export.h` (neatly sidestepping API vs ABI) in 1f23469a